Specification-Draft.MD

```diff
@@ -13,7 +17,31 @@ Examples of Desktop Agents include:
 - Autobahn
 - ThomsonReuters Eikon
 
-Desktop Agents expose an FDC3 standard API to applications in their own domain.  
+Desktop Agents expose an FDC3 standard API to applications they have launched.  When an App is launched by a Desktop Agent and is given access to the Agent's API to interoperate, it is running in that Desktop Agent's *context*. 
```

Comment:
--D--esktop ... is this some sort of import issue? lots of your paragraphs are missing a letter!

Reply:
I don't agree with making the Desktop Agent the central concept. 
I also don't agree that the central concept is that an App is launched by a platform and has to use the Desktop Agent to interact with FDC3. 
For example we have clients who produce their own Launchers, which are not run from a Platform.